views/pools/header/action-group.tsx

```diff
@@ -46,7 +46,7 @@ const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
             </Box>
           }
         >
-          Create Pool
+          Find Pool
```

Comment:
Changed to Create Pool

Reply:
We change to Find Pool... Figma not updated